features/README.md

```diff
@@ -1,5 +1,5 @@
 # Behaviour driven specificatie
-Het doel is door middel van gerichte voorbeelden de functionele werking van de api specificeren en illustreren. Voor de meeste features is de ontwikkelaar van de provider van de API doelgroep.
+Het doel is door middel van gerichte voorbeelden de functionele werking van de api te specificeren en te illustreren. Voor de meeste features is de ontwikkelaar van de provider van de API de doelgroep.
```

Comment:
Ik denk niet dat het correct is om te zeggen dat de ontwikkelaar van de provider de doelgroep is. De doelgroep is volgens mij alle stakeholders:
- de domein kenners kan in natuurlijke taal lezen wat de requirement is
- de provider developer kan bepalen waaraan zijn implementatie moet voldoen
- de consumer developer kan bepalen wat hij bij een gegeven input als output kan verwachten
- de tester kan bepalen of de implementatie voldoet aan de requirement
- enz

Reply:
Voorstel voor een nieuwe tekst:
De features specificeren en illustreren de (functionele) werking van de API met gerichte voorbeelden.
Zo is voor alle stakeholders duidelijk wat de API doet en hoe de geleverde informatie tot stand komt. Iedere stakeholder kan de feature files gebruiken,  maar ook bijdragen aan verbetering van de functionaliteit:

- de domeinkenner kan in natuurlijke taal lezen wat de requirement is
- de provider developer kan bepalen waar zijn implementatie aan moet voldoen
- de consumer developer kan bepalen wat hij bij een gegeven input als output kan verwachten
- de tester kan bepalen of de implementatie voldoet aan de requirement
- enz.

Reply:
Lijkt me een hele goede verbetering, ik heb hem aangepast.